src/Components/GrowthChart/GrowthChartBuilder/GrowthChartBuilder.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { Line } from 'react-chartjs-2';
+import 'chart.js/auto';
+import { ChartData } from '../../../types/chartDataTypes';
+import { useRangeTimePeriode } from './useRangeTimePeriode';
+
+interface GrowthChartBuilderProps {
+    chartData?: ChartData;
+}
+
+export const GrowthChartBuilder = ({ chartData }: GrowthChartBuilderProps) => {
+    const dataSetName = Object.keys(chartData.datasets)[0];
+    const dataSet = chartData.datasets[dataSetName];
+    const dataSetMetadata = chartData.metadata[dataSetName];
```

Comment:
Should we maybe move this logic to the GrowthChart.tsx file and then send dataSet and dataSetMetaData to this file as a prop?
This would clean up the code in the ChartBuilder file